src/main/java/org/homework/Status.java

```diff
@@ -0,0 +1,5 @@
+package org.homework;
+
+public enum Status {
+    완료, 미완료
+}
```

Comment:
열거형을 한글로 작성해도 문제는 없지만, 보통 영어로 작성합니다.
따로 변수를 둬서 한글을 입력받을 수 있도록 수정해주세요~
```suggestion
public enum Status {
    COMPLETED("완료"),
    INCOMPLETE("미완료");
    private final String description;
    Status(String description) {
        this.description = description;
    }
    public String getDescription() {
        return description;
    }
}
```